### PR TITLE
feat: Sunset gov v1

### PR DESCRIPTION
**Pull request by vgantchev**
This PR sunsets gov v1 and the Council, more specifically:
* Removes the following pallets from the runtime: `elections-phragmen`, `tips`, `collectives/council`
* Adds migrations to the locks on funds and clean the storage associated with the pallets above

`pallet-democracy` is left in the runtime for now, to be cleaned up later once the existing votes have expired (another max ~3 months)

### CODE REVIEW

File: runtime/basilisk/src/lib.rs [hunk 1]
```diff
@@ -170,13 +171,9 @@ construct_runtime!(
 		Treasury: pallet_treasury = 4,
 		Utility: pallet_utility = 5,
 		//NOTE: 6 - is used by Scheduler which must be after cumulus_pallet_parachain_system
-		Democracy: pallet_democracy exclude_parts { Config } = 7,
```
Comment (mrq1911) [inline runtime/basilisk/src/lib.rs:173]: we should probably retire those pallet id's of removed pallets

**Review by mrq1911 — APPROVED** [review]
fmt & bump versions & LGMT

File: runtime/basilisk/src/lib.rs (hunk 1 above)
Comment (vgantchev) [reply runtime/basilisk/src/lib.rs:173]: wdym by retire the pallet IDs?

Comment (mrq1911) [reply runtime/basilisk/src/lib.rs:173]: not use them in future

**Review by mrq1911 — APPROVED** [review]

File: runtime/basilisk/src/lib.rs [hunk 2]
```diff
@@ -171,12 +172,9 @@ construct_runtime!(
 		Utility: pallet_utility = 5,
 		//NOTE: 6 - is used by Scheduler which must be after cumulus_pallet_parachain_system
 		Democracy: pallet_democracy exclude_parts { Config } = 7,
-		Elections: pallet_elections_phragmen = 8,
```
Comment (jak-pan) [inline runtime/basilisk/src/lib.rs:174]: We should check the storage after we remove pallets, I think we'll need to clear it either by ref or by migration.

File: runtime/basilisk/src/governance/old.rs [hunk 3]
```diff
@@ -19,29 +19,12 @@ use crate::governance::TechnicalCollective;
 use crate::*;
 
 use primitives::constants::{
-	currency::{CENTS, DOLLARS, UNITS},
+	currency::DOLLARS,
```
Comment (jak-pan) [inline runtime/basilisk/src/governance/old.rs:22]: Why remove this and then use imports directly? Just change to primitives if it changed?

File: runtime/basilisk/src/lib.rs (hunk 2 above)
Comment (jak-pan) [reply runtime/basilisk/src/lib.rs:174]: Ok, I see later we do migration

**Pull request merged**